src/merge.py

```
@@ -70,11 +70,11 @@ class EventAction(Enum):
 
 @dataclass
 class MergeEvent(object):
```

Ruff (0.15.9)

[warning] Class `MergeEvent` inherits from `object`

Remove `object` inheritance

(UP004)